CI/travis.before_install.sh

```diff
@@ -1,8 +1,4 @@
 #!/bin/bash
-if [ -z "${TRAVIS_OS_NAME}" ] || [ "${TRAVIS_OS_NAME}" = "linux" ]; then
-  echo Before install on linux.
```

Comment:
If I go looking into why a build failed or succeeded for a particular combination of build conditions on Travis it is helpful if I can compare how various stages in the process progressed - however as things stand the MacOs builds will report in the logs that they reached a certain stage whereas the Linux builds will now be silent at the same point.  I agreed that we can delete the scripts without a concern, but the text in the logs helps to decode where the problems might be (if there are any) - especially if the faults aren't Mudlet but Travis infrastructure related...! :scream: